include/net/xdp.h

```diff
@@ -63,13 +63,30 @@ struct xdp_rxq_info {
 	struct xdp_mem_info mem;
 } ____cacheline_aligned; /* perf critical, avoid false-sharing */
 
+#ifdef CONFIG_XDP_LUA
+struct xdplua_create_work {
```

Comment:
Maybe we should add a field of luadata's datarefs into this struct to automate the reference control of these objects

Reply:
@VictorNogueiraRio it makes sense to me.. also, we should rename this struct.. perhaps, `xdplua_work`.. `create_work` sounds like a function..